src/app/locationForm/thanks/heart.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Ha, this isn't just an image? :)

Reply:
It's an SVG. We are passing width/height through as params, so we still need React ;)